docs/analyzer/adding_recognizers.md

```diff
@@ -198,6 +198,71 @@ In both examples, the `/analyze` request is extended with a list of `ad_hoc_reco
 
 Additional examples can be found in the [OpenAPI spec](../api-docs/api-docs.html).
 
+### Reading pattern recognizers from YAML
+
+Recognizers can be loaded from a YAML file, which allows users to add recognition logic without writing code.
+Here's an example YAML file:
+
+``` yaml
+recognizers:
+  -
+    name: "Zip code Recognizer"
+    supported_language: "de"
+    patterns:
+      -
+         name: "zip code (weak)"
+         regex: "(\\b\\d{5}(?:\\-\\d{4})?\\b)"
+         score: 0.01
+    context:
+     - zip
+     - code
+    supported_entity: "ZIP"
+  -
+    name: "Titles recognizer"
+    supported_language: "en"
+    supported_entity: "TITLE"
+    deny_list:
+      - Mr.
+      - Mrs.
+      - Ms.
+      - Miss
+      - Dr.
+      - Prof.
+```
+
+Once the YAML file is created, it can be loaded into the `RecognizerRegistry` instance.
+
+This example creates a `RecognizerRegistry` holding only the recognizers in the YAML file:
+
+ <!--pytest-codeblocks:skip-->
+``` python
+from presidio_analyzer import AnalyzerEngine, RecognizerRegistry
+
+yaml_file = "recognizers.yaml"
+registry = RecognizerRegistry()
+registry.add_recognizers_from_yaml(yaml_file)
+
+analyzer = AnalyzerEngine(registry=registry)
+# analyzer.analyze(...)
```

Comment:
Why is this in comment?

Reply:
To illustrate that the next thing you'd do is call analyze. I didn't want to specify the parameters. If you think it's not clear enough I can just use the default parameters.

Reply:
I think it's preferable and not have comments in examples :)

Reply:
Agreed. Fixed!